docs/en/plugins/README.md

```diff
@@ -0,0 +1,413 @@
+
+```
+%outlet{name: “before-thread-title”}
+```
+
+You can find all of the existing outlets by searching for `%outlet` in the repository.
+
+_NB: You’ll need to pull request any new outlets against the main loomio repository, but don’t worry, we’ll definitely accept it if it’s named nicely and not right next to another outlet. We'll also be very excited to hear about what you're working on._
+
+##### 3. Making a component
+
+Now, we’ll write a little code to display tags next to the discussions title if they exist.
+
+```
+# plugins/loomio_tags/components/thread_tags/thread_tags.coffee
+angular.module('loomioApp').directive 'tags', ->
+  restrict: 'E'
+  replace: true
+  templateUrl: 'generated/components/tags.html'
+  controller: ($scope, Records) ->
+    $scope.tags = ->
+      Records.discussionTags.find(discussionId: $scope.thread.id)
+```
+
+Some things to note:
+
+- For now, plugin components must inherit the scope in which they inhabit. This means that if you put a plugin into an outlet in the ThreadPageController, then you'll have access to the ThreadPageController's $scope in your component. Here our outlet is in `thread_preview_collection.haml`, so we have access to the scope defined in `thread_preview_collection.coffee`
+- The template url format is `generated/components/<your_component_name>.html`
+- You can inject any class you'd like into the plugin controllers, which can be defined either in the core repo or within your plugin.
+- `Records` is a collection of all of our records interfaces, and `find` a method we can use to search for records within a given interface.
+
+```
+# plugins/loomio_tags/components/thread_tags/thread_tags.haml
+.thread-tags{ng-if: "anyTags()"}
+  .thread-tag{ng-repeat: "tag in tags()", style: "background-color: {{tag.color}};"}
+    %span {{tag.name}}
+```
+
+_NB: notice that these need to be in a folder within a /components folder in your plugin, and that that folder must be named the same as each file._
+
+Then, we can tell our plugin about the component we created, and what outlet we want it to live in, using the `use_component` command
+
+```
+plugin.use_component :thread_tags, outlet: :before_thread_title
+```
+
+_NB: We support using the same outlet for multiple components, so don’t worry about conflicts there._
+
+You can also put a single component into multiple outlets in the interface by passing an array of names:_
+
+```
+plugin.use_component :thread_tags, outlet: [:before_thread_title, :after_thread_title]
+```
+
+##### 4. Fetching the tags from the server
+The only thing left is to make an API call to fetch our tags! In order to do this, we'll create another directive, which specifically performs an API request when it's loaded. (This is the recommended way to load remote records into your app.)
+
+I've called it `tag_fetcher`, and it has just a single file, `tag_fetcher.coffee`
```

Comment:
I've called it `tag_fetcher`, and it has just a single file, `tag_fetcher.coffee`**:** [add colon]